spread/lxd.go

```diff
@@ -111,6 +126,13 @@ func (p *lxdProvider) Allocate(ctx context.Context, system *System) (Server, err
 	if !p.options.Reuse {
 		args = append(args, "--ephemeral")
 	}
+	if p.vm {
+		args = append(args, "--vm")
+	}
+	if p.backend.Memory > 0 {
+		mem := int(p.backend.Memory / mb)
+		args = append(args, "-c", fmt.Sprintf("limits.memory=%dMiB", mem))
+	}
```

Comment:
Does this work for both the VM and not VM backends? Probably worth mentioning in `README.md`.

Reply:
Yes, this applies to both containers and VMs. There's also CPU limit which can be set through `limits.cpu`, but that would require a field to be added to `Backend`. OTOH the QEMU backend could also support that with ease by passing `-smp <n>`, so maybe it's worth it.

Reply:
pushed an update to the README file